Clients/src/presentation/components/Modals/ComplianceFeedback/ComplianceFeedback.tsx

```diff
@@ -11,8 +11,38 @@ const AuditorFeedback: React.FC<AuditorFeedbackProps> = ({ activeSection }) => {
   const [file, setFile] = useState<File | null>(null);
 
   const handleFileUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
@melpsh 
Thank you for working the file upload logic. 🙏🏻
However, the logic must be separated from the UI.
Please move this function to **Application Layer** in **/tools/fileUtil.ts**.
Make it as flexible and customable possible and the import and use it here.
Thanks

Reply:
Hi Mohammad I created a new commit on the changes you asked I also have added an Error Modal for our error handling we can improve the Error Modal once we have the Figma design.